base_no_translation/model.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 initOS GmbH http://www.initos.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models
+
+
+class ModelExtended(models.Model):
+    _inherit = 'ir.model'
+
+    def _register_hook(self, cr):
+        for m in self.pool.models.itervalues():
+            if getattr(m, '_disable_field_translations', False):
```

Comment:
This is different from what the README says (and actually what I expect... So the README needs fixing :smile_cat:)